settings.gradle.kts

```diff
@@ -0,0 +1,59 @@
+rootProject.name = "hex-thai-star"
+
+include(":booking-api-out")
```

Comment:
wszystkie include umieszcza się zwyczajowo w pliku settings.gradle.kts

Reply:
to jest settings.gradle.kts